liquidcore/home/management/commands/createoauth2app.py

```diff
@@ -10,14 +10,24 @@ class Command(BaseCommand):
     def add_arguments(self, parser):
         parser.add_argument('app_name')
         parser.add_argument('redirect_uri')
+        parser.add_argument('--algo')
+        parser.add_argument('--noskip')
```

Comment:
these arguments look funny to me because you have an optional string and a present-override boolean defined in the same way - so you'd have to say `--algo X --noskip 1` instead of just `--algo X --noskip`

here's a more straightforward way of doing it:
https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/#s-accepting-optional-arguments

Reply:
noskip is now type as boolean, thks